ngraph/core/reference/include/ngraph/runtime/reference/deformable_convolution.hpp

```diff
@@ -14,31 +14,54 @@ namespace ngraph
         {
             namespace def_conv_impl
             {
-                inline void validate_params(const Shape& in_shape,
-                                            const Shape& o_shape,
-                                            const Shape& f_shape,
-                                            const Strides& strides,
-                                            const Strides& dilations,
-                                            const CoordinateDiff& pads_begin,
-                                            const CoordinateDiff& pads_end)
+                inline void validate_deformable_convolution_params(const Shape& in_shape,
+                                                                   const Shape& o_shape,
+                                                                   const Shape& f_shape,
+                                                                   const Shape& out_shape,
+                                                                   const Strides& strides,
+                                                                   const Strides& dilations,
+                                                                   const CoordinateDiff& pads_begin,
+                                                                   const CoordinateDiff& pads_end,
+                                                                   const int64_t& groups,
+                                                                   const int64_t& deformable_groups)
```

Comment:
you don't need to take basic types like int64_t by const reference here, regarding performance it's better to take them by value; read more in [CppCoreGuidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f16-for-in-parameters-pass-cheaply-copied-types-by-value-and-others-by-reference-to-const)

Reply:
Thanks for the tip. I passed them by value now